.rubocop.yml

```diff
@@ -30,6 +30,7 @@ RSpec/AnyInstance:
 RSpec/MultipleExpectations:
   Exclude:
   - 'spec/controllers/contribute_controller_spec.rb'
+  - 'spec/features/publication_workflow_spec.rb'
```

Comment:
I'm thinking of submitting a follow-up that simply excludes `spec/features/**/*` from this cop. I suspect the single example per spec rule just isn't a good fit for capybara `scenario` or feature specs in general.

Thoughts?